data-model/virtualcollections/virtualcollections.json

```diff
@@ -1 +1,99 @@
-
+{
+  "$schema": "http://json-schema.org/draft-2020-12/schema#",
+  "type": "object",
+  "properties": {
+    "ods:id": {
+      "type": "string",
+      "pattern": "^https:\\/\\/hdl\\.handle\\.net\\/20\\.5000\\.1025\\/.{3}-.{3}-.{3}$",
+      "examples": ["https://hdl.handle.net/20.5000.1025/XXX-XXX-XXX"]
+    },
+    "ods:type": {
+      "type": "string",
+      "description": "The FDO type of the object"
+    },
+    "ods:version": {
+      "type": "integer",
+      "description": "The version of the object."
+    },
+    "dcterms:creator": {
+      "type": "object",
+      "properties": {
+        "ods:type": {
+          "type": "string"
+        },
+        "foaf:name": {
+          "type": "string"
+        }
+      },
+      "description": "The creator of the collection."
+    },
+    "hasAccessRestrictions": {
```

Comment:
Missing prefix. Isn't this covered by one of the capabilities (or should it be covered there)?